lib/linguist/languages.yml

```diff
@@ -3894,6 +3894,15 @@ Ring:
   tm_scope: source.ring
   ace_mode: text
   language_id: 431
+Riot:
+  type: programming
+  color: "#A71E49"
+  group: HTML
```

Comment:
@crisward This is the reason for the [most recent build failure](https://travis-ci.org/github/linguist/jobs/332305128#L1388-L1390):

~~~
 1) Failure:
TestLanguage#test_no_unused_colours [/home/travis/build/github/linguist/test/test_language.rb:467]:
Unused colour assigned to Riot
~~~

Languages which are assigned a `group` will inherit their parent language's colour and usage stats, For example, Less [inherits CSS's name and colour too](https://github.com/jonikorpi/Less-Framework).

In other words, you'll want to either ditch the `group` or the colour field, one or the other. =)

Reply:
Thanks! I'll dump the group them.